### DISCUSSION TITLE
VecAlign

### DISCUSSION THREAD

**Discussion by jgcb00** [bitextor/bitextor#230] [Ideas]
Hi,
I saw that you forked VecAlign to your team repository,
Are you planning on adding this alignement method to bitextor ?

Regards,

**Comment by cgr71ii** [comment]
Hi,

We forked Vecalign in order to make a few changes (briefly, automatically embeddings generation and space optimization). This fork was used in [Bitextor Neural](https://github.com/bitextor/bitextor-neural). We are planning to integrate these changes directly to Bitextor. These changes might be integrated for Bitextor >= 8.2

If you are planning to use Bitextor Neural, be aware that newest changes in Bitextor, likely, will not be added. This is because, as I have said, we will integrate Bitextor Neural' changes to Bitextor.

**Reply by jgcb00** [reply]
Okay thanks, great new feature to  bitextor then ! 
Good luck !

**Comment by cgr71ii** [comment]
Hi!

Vecalign (and [Neural Document Aligner](https://github.com/bitextor/neural-document-aligner/), a.k.a. NDA) has been integrated and are available in master branch. The changes will be available in the next Bitextor release, but in case that someone wants to test it before, you can [install manually Bitextor](https://github.com/bitextor/bitextor/blob/master/docs/INSTALL.md#manual-installation).

Please, if you test the new integrated neural tools, let us know if you find something that doesn't work as should creating an issue.
